pos_partner_location_map/models/res_partner.py

```diff
@@ -0,0 +1,7 @@
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    map_partner_address = fields.Char()
```

Comment:
This field might be misleading for a user. Because there are core Odoo fields used for the same goal.
We should parse this string into Odoo address fields instead.

Reply:
Resolved